.github/workflows/create_update_issue.yml

```diff
@@ -0,0 +1,31 @@
+name: Create Fidesdemo Update Issue
+
+on:
+  repository_dispatch:
+    types: [new-fidesops-release]
+
+jobs:
+  Create-Issue:
+    name: Create Fidesops Update Issue
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Issue
+        uses: imjohnbo/issue-bot@v3
```

Comment:
We ought to be careful here which bots we give permission to write to our repos. @PSalant726 / @eastandwestwind did you vet this at all?

Reply:
no, I didn't vet this, I copied over much of this code from fides plus

Reply:
This just wraps the official GitHub APIs, and it's referenced in the official GitHub Actions docs, here: https://docs.github.com/en/actions/managing-issues-and-pull-requests/scheduling-issue-creation